meson_mesa3d.mk

```diff
@@ -13,6 +13,7 @@ include $(LOCAL_PATH)/aospext_cleanup.mk
 AOSPEXT_PROJECT_NAME := MESA3D
 AOSPEXT_BUILD_SYSTEM := meson
 
+MESA3D_VERSION = $(shell cat $(BOARD_MESA3D_SRC_DIR)/VERSION | cut -d '.' -f 1-2)
```

Comment:
Hi @maurossi , thank you for your contribution.

Would it be possible to check the version in a single place and set the variables to use later? E.g.:

```make
MESA3D_GALLIUM_LIBDIR :=
MESA3D_POPULATE_SYMLINKS :=

ifeq ($(shell expr $(MESA3D_VERSION) \<= 24.1), 1)
MESA3D_GALLIUM_LIBDIR := dri
MESA3D_POPULATE_SYMLINKS := true
endif
```